crates/tracing/src/lib.rs

```diff
@@ -114,18 +114,21 @@ pub fn configure_tracing(
 
 #[cfg(feature = "otel")]
 #[allow(clippy::missing_errors_doc)] // TODO: Document errors
+#[allow(clippy::needless_pass_by_value)]
```

Comment:
Note that we discussed this offline and decided to allow this instead of making a breaking API change at this time, but this is worth removing and resolving whenever we move the wasmcloud-tracing crate to 0.2.0 for legitimate reasons 😄